plugins/woocommerce/client/blocks/package.json

```diff
@@ -210,7 +210,7 @@
 		"ignore-loader": "0.1.x",
 		"jest": "29.5.x",
 		"jest-circus": "29.5.x",
-		"jest-fixed-jsdom": "0.0.9",
+		"jest-fixed-jsdom": "^0.0.10",
```

Comment:
Hint: we can also pin this version in `.syncpackrc` so future bumps are uniformly applied across the monorepo.